services/introspection/local.go

```diff
@@ -105,8 +106,18 @@ func (l *Local) Server(ctx context.Context, _ *ptypes.Empty, _ ...grpc.CallOptio
 	if err != nil {
 		return nil, errdefs.ToGRPC(err)
 	}
+	pid := os.Getpid()
+	var pidns uint64
+	if runtime.GOOS == "linux" {
```

Comment:
nit: is this `if` block necessary since you have an (empty) implementation of `statPIDNS` for all non-Linux platforms?

Reply:
Not necessary, but added for readability